### DISCUSSION TITLE
Iron Bank Repo

### DISCUSSION THREAD

**Discussion by thermoscookies** [NUWCDIVNPT/stig-manager#769] [General]
Hello,

In your docker images. there is a tag 'latest-ironbank' I had hoped that meant this package was available within the ironbank repo but when I searched I wasn't able to find it.  Does it happen to be there but I don't have access to it or does there happen to be a plan to have it available there?

Thank you for your time! Hoping to be able to make use of this wonderful package that you have put together!

**Comment by cd-rite** [comment]
Hi @thermoscookies 
At the moment, we do not have the resources to meet the process requirements for being hosted on Iron Bank. 
However, as you noticed, we do offer an image that is based on the Iron Bank NodeJS image.  If you don't want to use that one, you could build the container yourself from our Dockerfile, passing the Iron Bank base image as a build argument.  
We currently use this image: `registry1.dso.mil/ironbank/opensource/nodejs/nodejs16:latest`

Hope that helps! Let us know how it goes!

**Comment by cd-rite** [comment]
HI @thermoscookies 
I checked into it again, and it seems Iron Bank has drastically reduced the barrier to entry to included in their registry, so we are exploring this option again. Thanks for bringing it back to our attention!

**Comment by cd-rite** [comment]
@thermoscookies Forgot to update this thread, but we are now hosted on Iron Bank:
registry1.dso.mil/ironbank/opensource/stig-manager
Thanks for reminding us of it a few months ago!
